atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -384,10 +384,10 @@ private void lowerConsistencyWhenSafe() {
         try {
             dcs = clientPool.runWithRetry(client ->
                     CassandraVerifier.sanityCheckDatacenters(
-                            client,
+                            client.rawClient(),
```

Comment:
Unrelated to this PR, but as this makes me sad:
@fsamuel-bs This is super funky. Why does the Cassandra client not support ``describe_keyspace`` and some other methods. This will make the interface very confusing to use especially as I can start calling ``client.rawClient().multiget_slice`` etc. Can we add all methods we expect to be used in AtlasDB to the interface?